src/CoolPropLib.def

```diff
@@ -15,6 +15,8 @@ EXPORTS
   AbstractState_update_and_1_out = _AbstractState_update_and_1_out@40
   AbstractState_update_and_5_out = _AbstractState_update_and_5_out@56
   AbstractState_update_and_common_out = _AbstractState_update_and_common_out@52
+  AbstractState_get_fugacity = _AbstractState_get_fugacity@20
+  AbstractState_get_fugacity_coefficient = _AbstractState_get_fugacity_coefficient@20
```

Comment:
I am also wondering if we need to add export lines to the `CoolPropLib.def` file. I noticed some functions in `CoolPropLib.cpp` that are missing from the `.def` file, but if it is required, can you check the integer value used after the `@`? I believe it is 4 bytes per arg

Reply:
Hmm good question, I don't think I have touched that in ages. Can you check if anywhere in the CMakeLists.txt references this def file? I think it is not used anymore...

Reply:
There is not a reference to the `.def` file in the the file you mention. I will go ahead and remove this. Other functions written within `CoolPropLib.cpp` are left out of the `.def` file and work properly already.

Reply:
I have removed them now, so I am resolving this thread